pixi.toml

```diff
@@ -84,6 +84,9 @@ typos = "*"
 pre-commit-install = "pre-commit install"
 pre-commit-run = "pre-commit run -a"
 
+[feature.py310.dependencies]
+python = "3.10.*"
```

Comment:
Don't we need to add `typing-extensions` as additional dependency for `py310`?

Reply:
[fc59ee6](https://github.com/Quantco/dataframely/pull/88/commits/fc59ee69eb2dd7c9d84e8e5c303df874771f01d9)